packages/compass/src/main/window-manager.ts

```diff
@@ -43,7 +43,7 @@ const DEFAULT_HEIGHT = (() => {
   return height;
 })();
 
-const MIN_WIDTH = process.env.COMPASS_MIN_WIDTH ?? 1024;
+const MIN_WIDTH = process.env.COMPASS_MIN_WIDTH ?? 1025;
```

Comment:
How do you feel about leaving a comment about why we have the min width set at that amount?
```suggestion
// We set the min width to 1025 so that the screensize breakpoints of leafygreen
// components are not hit. The breakpoints make the styles of the Select component
// change significantly at widths of 1024 and less.
const MIN_WIDTH = process.env.COMPASS_MIN_WIDTH ?? 1025;
```